README.md

```diff
@@ -167,6 +178,10 @@ The Photoshop APIs currently support creating and editing of Embedded Smart Obje
 - In order to update an embedded smart object that is referenced by multiple layers you only need to update one of those layers, the effect will be reflected in all layers referencing the same smart object.
 
 Please see the api docs for more information.
+https://git.corp.adobe.com/pages/dice/pie-in-the-sky/#api-Photoshop-document_operations
```

Comment:
@khound - same private link problem

Reply:
I would also just make this a link:
"[API Documentation](https://adobedocs.github.io/photoshop-api-docs/#api-Photoshop-document_operations)

Reply:
Good suggestion.changing both.